tests/plugins/stylescombo/listreuse.js

```diff
@@ -53,19 +53,32 @@
 
 			assert.areEqual( '', stylesCombo._.value );
 			assert.isFalse(stylesCombo._.list.isMarked('ol-a'));
+
+			stylesCombo.onClick( 'ol-a' );
+			assert.areSame( '<ol class="ol-a"><li><ol class="ol-a"><li>&nbsp;</li></ol></li></ol>', bot.getData( true ) );
 		},
 		'test list style value is not selected from parent list item when in non-list child': function() {
 			var editor = this.editor,
 				stylesCombo = editor.ui.get( 'Styles'),
 				bot = this.editorBot;
 
-			bot.setHtmlWithSelection( '<ol class="ol-a"><li><ol><li><strong>^</strong></li></ol></li></ol>' );
+			bot.setHtmlWithSelection( '<ol class="ol-a"><li><ol><li><strong>asd^</strong></li></ol></li></ol>' );
 
 			stylesCombo.createPanel( editor );
 			stylesCombo.onOpen();
 
 			assert.areEqual( '', stylesCombo._.value );
-			assert.isFalse(stylesCombo._.list.isMarked('ol-a'))
+			assert.isFalse(stylesCombo._.list.isMarked('ol-a'));
```

Comment:
added assertions on this afterwards.  no, the marked changes when you apply / unapply a style